package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "apify-cli",
-    "version": "0.21.4",
+    "version": "0.21.5",
```

Comment:
why is that? we havent shipped 0.21.4 to stable yet, right?

Reply:
oh true, I forgot 💀